lib/utils.js

```diff
@@ -0,0 +1,16 @@
+
+var isObject = function isObject(value) {
+    return value instanceof Object && value.constructor === Object;
+};
+
+var isArray = function isArray(value) {
```

Comment:
I thought all versions of node.js have `Array.isArray`.  Is there some other context where `node-cache-manager` could be used where `Array.isArray` is not defined?

Reply:
You're totally right, I will remove the helper.